modules/apps/object/object-web/src/main/resources/META-INF/resources/js/modules/global.d.ts

```diff
@@ -12,21 +12,6 @@
  * details.
  */
 
```

Comment:
Where is this file referenced? any `global.d.ts` should also be referenced in our [globals file ](https://github.com/liferay/liferay-portal/blob/master/modules/global.d.ts)

Reply:
I didn't know that, thanks for that lesson! I thought `global.d.ts` where intent to be visible into the module and not across the portal.

I think this definitions should not be available across the portal, so probably a better approach can be rename that to index.d.ts. Am I right?

Reply:
Yeah that is right, if the types are attached to the Liferay global object, then it shoudl be added to our global.d.ts, but if it is just local to that module, then you can create an index.d.ts